hw1/02-palindrome.js

```diff
@@ -1,3 +1,19 @@
-const elem = document.querySelector('input');
+const elem = document.querySelector("input");
 
-elem.addEventListener('input', handleInput);
+elem.addEventListener("input", handleInput);
```

Comment:
Looks like all your strings use double quotes, this is good! I like the consistency.